linode/manager
# M3-5479: Premium LKE create page UI refinement

## Description

Cleans up the `KubeCheckoutBar` and `EnhancedNumberInput` components. The overall changes should be:

- [ ] Cluster Summary head (smaller, no green color)

- [ ] Left margin of summary sidebar

- [ ] Rule color (it's a darker gray color, new spec established for Linode Create page)

- [ ] Spacing between rule and Linode Plan name

- [ ] New 'minus' icon (shorter now), svg attached

- [ ] New 'plus' icon, svg attached

- [ ] Plus / minus controls

- [ ] Changed for both Plans table and summary sidebar—1. Box removed, new hover state, 2. Price text (smaller, no green color)

## How to test

Navigate to the Kubernetes cluster creation page and ensure the changes specified are shown.
